controllers/clusterversion_controller.go

```diff
@@ -102,6 +103,10 @@ func (r *ClusterVersionReconciler) ensureConsolePlugin(clusterVersion string) er
 	// Create/Update ODF console ConfigMap (nginx configuration)
 	odfConsoleConfigMap := console.GetNginxConfConfigMap(OperatorNamespace)
 	_, err = controllerutil.CreateOrUpdate(context.TODO(), r.Client, odfConsoleConfigMap, func() error {
+		if odfConsoleConfigMapData := odfConsoleConfigMap.Data["nginx.conf"]; odfConsoleConfigMapData != nginxConf {
```

Comment:
Instead of an if why can not we always assign the value. It won't have any effect, right?

Reply:
Resolving it, we sorted it over chat